src/RageInput.cpp

```diff
@@ -60,6 +65,11 @@ void RageInput::LoadDrivers()
 	m_InputHandlers.clear();
 	g_mapDeviceToHandler.clear();
 
+#if LINUX
+	// Recreating this forces it to re-scan for devices.
+	LINUXINPUT = new LinuxInputManager;
```

Comment:
This seems like memory leak since we're not freeing the previous version if one exists. Needs to be addressed.